GooglePayment/src/main/java/com/braintreepayments/api/GooglePayment.java

```diff
@@ -198,6 +198,12 @@ public static void requestPayment(final BraintreeFragment fragment, final @NonNu
             @Override
             public void onConfigurationFetched(Configuration configuration) {
 
+                if (!configuration.getGooglePayment().isEnabled(fragment.getApplicationContext())) {
+                    fragment.postCallback(new BraintreeException("This merchant does not have Google Pay enabled," +
```

Comment:
```suggestion
                    fragment.postCallback(new BraintreeException("Google Pay enabled is not enabled for your Braintree account," +
```

Reply:
suggestion to the language to be more focused towards the merchant and not BT support